Azure Devops Pipeline failing on Nuget restore with latest bot-solutions package 1.1.0

#### What project is affected? Virtual Assistant

#### What language is this in? .net

#### What happens?
when using the sample [here](https://github.com/microsoft/botframework-solutions/tree/master/samples/csharp/assistants/virtual-assistant) the build pipeline fails with the following error
##[error]The nuget command failed with exit code(1) and error(F:\agents\_work\_tool\dncs\2.2.100\x64\sdk\2.2.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 3.1.  Either target .NET Core 2.2 or lower, or use a version of the .NET SDK that supports .NET Core 3.1. 

#### What are the steps to reproduce this issue?
build new template using latest version
use pipeline example that is included in the template
push project to Azure Devops and let the build pipeline run

#### What were you expecting to happen?
The build to succeed and the project is built
#### Can you share any logs, error output, etc.?
see above error
#### Any screenshots or additional context?

--- a/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Pipeline/VirtualAssistantSample.yml
+++ b/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Pipeline/VirtualAssistantSample.yml
@@ -60,7 +60,8 @@ steps:
 - task: CopyFiles@2
   displayName: 'Copy VA'
   inputs:
-    SourceFolder: '$(System.DefaultWorkingDirectory)\templates\Virtual-Assistant-Template\csharp\Sample\VirtualAssistantSample'
+    # if your working directory is not root, you may change the following path
+    SourceFolder: '$(System.DefaultWorkingDirectory)\VirtualAssistantSample'
     Contents: '**\*'
     TargetFolder: '$(Build.ArtifactStagingDirectory)\VA'
 

--- a/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Pipeline/VirtualAssistantSample.yml
+++ b/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Pipeline/VirtualAssistantSample.yml
@@ -57,7 +57,7 @@ steps:
     summaryFileLocation: '$(Build.SourcesDirectory)\VirtualAssistantSample.Tests\coverage.cobertura.xml'
     reportDirectory: '$(Build.SourcesDirectory)\VirtualAssistantSample.Tests'
 
- - task: CopyFiles@2
+- task: CopyFiles@2
   displayName: 'Copy VA'
   inputs:
     SourceFolder: '$(System.DefaultWorkingDirectory)\templates\Virtual-Assistant-Template\csharp\Sample\VirtualAssistantSample'

--- a/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Pipeline/VirtualAssistantSample.yml
+++ b/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Pipeline/VirtualAssistantSample.yml
@@ -1,31 +1,28 @@
 # specific branch build
 trigger:
-  branches:  
+  branches:
     include:
     - master
     - feature/*
-  
+
 pool:
-   name: Hosted VS2017
-   demands:
-    - msbuild
-    - visualstudio
+   vmimage: 'windows-2019'
 
 variables:
   buildPlatform: 'Any CPU'
   buildConfiguration: 'Release'
 
 steps:
-- task: DotNetCoreInstaller@0
-  displayName: 'Use .NET Core sdk 2.2.100'
+- task: DotNetCoreInstaller@2
+  displayName: 'Use .NET Core sdk 3.1'
   inputs:
-    version: 2.2.100
+    version: 3.1.x
   continueOnError: true
 
 - task: NuGetToolInstaller@0
-  displayName: 'Use NuGet 4.9.1'
+  displayName: 'Use NuGet 5.3.0'
   inputs:
-    versionSpec: 4.9.1
+    versionSpec: 5.3.0
 
 - task: NuGetCommand@2
   displayName: 'NuGet restore'

--- a/templates/csharp/VA/VA/Pipeline/VirtualAssistantSample.yml
+++ b/templates/csharp/VA/VA/Pipeline/VirtualAssistantSample.yml
@@ -1,31 +1,28 @@
 # specific branch build
 trigger:
-  branches:  
+  branches:
     include:
     - master
     - feature/*
-  
+
 pool:
-   name: Hosted VS2017
-   demands:
-    - msbuild
-    - visualstudio
+   vmimage: 'windows-2019'
 
 variables:
   buildPlatform: 'Any CPU'
   buildConfiguration: 'Release'
 
 steps:
-- task: DotNetCoreInstaller@0
-  displayName: 'Use .NET Core sdk 2.2.100'
+- task: DotNetCoreInstaller@2
+  displayName: 'Use .NET Core sdk 3.1'
   inputs:
-    version: 2.2.100
+    version: 3.1.x
   continueOnError: true
 
 - task: NuGetToolInstaller@0
-  displayName: 'Use NuGet 4.9.1'
+  displayName: 'Use NuGet 5.3.0'
   inputs:
-    versionSpec: 4.9.1
+    versionSpec: 5.3.0
 
 - task: NuGetCommand@2
   displayName: 'NuGet restore'
@@ -60,10 +57,11 @@ steps:
     summaryFileLocation: '$(Build.SourcesDirectory)\$safeprojectname$.Tests\coverage.cobertura.xml'
     reportDirectory: '$(Build.SourcesDirectory)\$safeprojectname$.Tests'
 
- - task: CopyFiles@2
+- task: CopyFiles@2
   displayName: 'Copy VA'
   inputs:
-    SourceFolder: '$(System.DefaultWorkingDirectory)\templates\Virtual-Assistant-Template\csharp\Sample\VirtualAssistantSample'
+    # if your working directory is not root, you may change the following path
+    SourceFolder: '$(System.DefaultWorkingDirectory)\$safeprojectname$'
     Contents: '**\*'
     TargetFolder: '$(Build.ArtifactStagingDirectory)\VA'
 

